docs/_documentations/remotedeploy-projects-eclipse.md

```diff
@@ -26,23 +26,24 @@ Select the template type and enter a project name. Your new project is built and
 
 Copy an existing local project, for example `myFirstNodeProject`, over to the remote Codewind deployment:
 
-1\. Select the remote deployment and click `Add Existing Project`:
+1\. Right-click on your remote connection and select **Add Existing Project**:
 
 ![Add existing project](./images/remotedeploy-projects-eclipse/eclipse_add-project.png){:width="366"}
 
-2\. Go to the folder that contains the local project files, click `Select a project from the workspace`, and select your project: 
+2\. Click **Select a project from the workspace**, and select your project: 
 
 ![Add to Codewind](./images/remotedeploy-projects-eclipse/eclipse_select-project.png){:width="654"}
 
-Codewind prompts you to confirm the project type, in this case, identified as `NodeJS`:
+Codewind prompts you to confirm the project type, in this case, identified as `Node.js`. Confirm the project type and click **Finish**:
 
 ![Confirm Project Type](./images/remotedeploy-projects-eclipse/eclipse_confirm-project.png){:width="654"}
 
-If you add a project to the remote connection that is already deployed on the local connection, the **Project Already Deployed** page appears: 
+If you add a project to the remote connection that is already deployed on the local connection, the **Project Already Deployed** dialogue appears: 
```

Comment:
Regarding dialogue and dialog, this is from Acrolinx:
- Dialogue: Use only in technical information to refer to a secondary window that opens and that gathers additional information from the user (for example, a "Save as" dialog box). Do not use "dialog" for "dialogue box."
- Dialog: Spell as shown to refer to communication with a computer—not "dialogue." For example, write "The terminal service recorder recorded the dialog between the user and the application." Spell as "dialogue" only to refer to a conversation between people. Do not use "dialog" for "dialog box." Use "dialog box" only in technical information for programmers, where it is important to distinguish the different parts of the interface. For most user information, use "window" when referring to a graphical interface.

Sounds like "window" might be the safest option here?